x-pack/plugins/enterprise_search/public/applications/shared/tables/reorderable_table/body_row.tsx

```diff
@@ -46,6 +48,15 @@ export const BodyRow = <Item extends object>({
           </EuiFlexGroup>
         </EuiFlexItem>
       </EuiFlexGroup>
+      {errors.length > 0 && (
+        <EuiFlexGroup direction="column">
+          {errors.map((errorMessage, errorMessageIndex) => (
+            <EuiFlexItem key={errorMessageIndex}>
+              <EuiCallOut iconType="alert" size="s" color="danger" title={errorMessage} />
```

Comment:
I see the alert icon here, but not in the screenshot. I'm assuming the screenshot is just out of date, yeah?

Reply:
Whoops! Yes!

![Screen Shot 2021-11-09 at 1 37 02 PM](https://user-images.githubusercontent.com/2479295/141169778-1528201b-fc31-49fc-bda7-e7499b4ed477.png)
 
Sorry, used the old screenshot